packages/client/hmi-client/src/components/model/model-parts/tera-model-part-entry.vue

```diff
@@ -125,71 +90,63 @@ const props = defineProps<{
 
 const emit = defineEmits(['update-item']);
 
-const nameText = ref(props.name);
-const unitExpression = ref(props.unitExpression);
-const descriptionText = ref(props.description);
-const showDescription = ref(!!descriptionText.value);
-
-const query = ref('');
-const results = ref<DKG[]>([]);
+const nameText = computed({
+	get: () => props.name,
+	set: (newName) => emit('update-item', { key: 'name', value: newName })
+});
```

Comment:
does this pattern work with enriching as well as editing?

Reply:
Yes, it's the same patern, enriching is done on the back-end, not on the front-end.